### PR TITLE
Milestone11: projects Popup window 

### PR DESCRIPTION
**Pull request by RuriEnciso**
### **Milestone11: projects Popup window** 

In this milestone we: 
- Reduced objects in script.js to only one.
- Deleted the second popups object
- Fixed linters

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -17,45 +17,87 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
...
     languages: ['Ruby on Rails', 'CSS', 'Javascript', 'HTML'],
     buttonName: 'See Project',
+
+    image: './images/desktop/Snapshoot-Portfolio.svg',
+    titlePopup: 'Keeping track of hundreds of components',
+    languagesPopup: ['HTML', 'CSS', 'Javascript'],
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.",
+    liveLink: 'https://www.example.com/',
+    sourceLink: 'https://github.com/RuriEnciso/Portfolio-Project',
   },
```
Comment (mwanawabangona) [inline script.js:45]: - [ ] `required`
- [ ] Amazing job so far! 👏 👏 
- [ ] Kindly note ❗ this applies to all your pops mobile pop cards included
- [ ] Please add a blur background on all the cards just as the template
- [ ] please see screenshots below.

`original template`
image

`your template`
image

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @RuriEnciso ,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -248,7 +248,7 @@ nav {
   height: 100vh;
   width: 100vw;
   overflow: auto;
-  background-color: rgba(0, 0, 0, 0.5);
+  background-color: rgba(193, 199, 208);
```
Comment (andersonlebon) [inline style.css]: - [ ] Kindly make sure the backend is blur just like the previous reviewer said. User this property to help you `backdrop-filter: blur(12px);`

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @RuriEnciso 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @RuriEnciso, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**